docs/documenting/yaml-style-guide.md

```diff
@@ -233,10 +233,10 @@ list `[]` by default.
 
 # Bad
 - alias: "Test"
-    triggers:
-      -  trigger: state
-         entity_id: binary_sensor.motion
-    condition: []
+  triggers:
```

Comment:
Can you also fix the `# Good` example just above?

Reply:
Done.